akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -204,6 +204,17 @@ object Source {
     })
 
   /**
+   * Create [[Source]] that will continually produce given elements in specified order.
+   *
+   * Start a new 'cycled' `Source` from the given elements. The producer stream of elements
+   * will continue infinitely by repeating the same sequence of elements provided during construction.
```

Comment:
sorry for nitpicking, but this comment is not completely true, it's not necessarily the same sequence of elements. For each round the `f` is evaluated and it doesn't have to return the same thing each time